{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@mitre.org",
			"ID": "CVE-2023-25399"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "A refcounting issue which leads to potential memory leak was discovered in scipy commit 8627df31ab in Py_FindObjects() function. Note: This is disputed as a bug and not a vulnerability. SciPy is not designed to be exposed to untrusted users or data directly."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "NVD-CWE-Other"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://github.com/scipy/scipy/issues/16235",
					"refsource": "MISC",
					"tags": [
						"Exploit",
						"Issue Tracking",
						"Patch",
						"Vendor Advisory"
					],
					"url": "https://github.com/scipy/scipy/issues/16235"
				},
				{
					"name": "https://github.com/scipy/scipy/pull/16397",
					"refsource": "MISC",
					"tags": [
						"Patch"
					],
					"url": "https://github.com/scipy/scipy/pull/16397"
				},
				{
					"name": "http://www.square16.org/achievement/cve-2023-25399/",
					"refsource": "MISC",
					"tags": [
						"Third Party Advisory"
					],
					"url": "http://www.square16.org/achievement/cve-2023-25399/"
				},
				{
					"name": "https://github.com/scipy/scipy/issues/16235#issuecomment-1625361328",
					"url": "https://github.com/scipy/scipy/issues/16235#issuecomment-1625361328"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "LOCAL",
				"availabilityImpact": "HIGH",
				"baseScore": 5.5,
				"baseSeverity": "MEDIUM",
				"confidentialityImpact": "NONE",
				"integrityImpact": "NONE",
				"privilegesRequired": "LOW",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
				"version": "3.1"
			},
			"exploitabilityScore": 1.8,
			"impactScore": 3.6
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:scipy:scipy:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "1.10.0",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-07-03T01:39Z",
	"publishedDate": "2023-07-05T17:15Z"
}
